server/verify.go

```diff
@@ -245,7 +245,7 @@ func validateAKPub(ak crypto.PublicKey, opts VerifyOpts) error {
 
 func validateAKCert(akCert *x509.Certificate, opts VerifyOpts) (*pb.MachineState, error) {
 	if len(opts.TrustedRootCerts) == 0 {
-		return nil, validateAKPub(akCert.PublicKey.(crypto.PublicKey), opts)
+		return &pb.MachineState{}, validateAKPub(akCert.PublicKey.(crypto.PublicKey), opts)
```

Comment:
Please add a test case for this and ideally check that it fails and then passes after changing the code.

Reply:
added